src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -771,6 +771,7 @@ void ov::npuw::CompiledModel::implement_properties() {
                           BIND(npuw::parallel_compilation, NPUW_PARALLEL_COMPILE),
                           BIND(npuw::partitioning::dcoff_type, NPUW_DCOFF_TYPE),
                           BIND(npuw::partitioning::dcoff_with_scale, NPUW_DCOFF_SCALE),
+                          BIND(npuw::partitioning::transpose_weights, NPUW_DCOFF_SCALE),
```

Comment:
Hello, it is typo here: instead of `NPUW_DCOFF_SCALE` there should be `NPUW_TRANSPOSE_WEIGHTS`

Reply:
Thank you changing it.